integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/base/ITAbstractUITest.java

```diff
@@ -0,0 +1,9 @@
+package to.etc.domui.webdriver.core.base;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import to.etc.domui.webdriver.core.AbstractWebDriverTest;
+
+@NonNullByDefault
+public class ITAbstractUITest extends AbstractWebDriverTest {
```

Comment:
Never create a base class if you have nothing to put in there. They tend to remain forever, and it never saves any reasonable time- it is quick enough to add them later.